src/main/com/fulcrologic/rad/routing/html5_history.cljc

```diff
@@ -86,12 +86,14 @@
    parameter is provided the mode is autodetected from presence of hash segment in URL.
   "
   ([] (url->route #?(:clj  false
-                     :cljs (some? (seq (.. js/document -location -hash))))))
-  ([hash-based?]
+                     :cljs (some? (seq (.. js/document -location -hash)))) nil))
+  ([hash-based? prefix]
```

Comment:
Sorry, this is a breaking change. You're causing users of arity one to suddenly need more parameters. If you need an additional parameter here, add an additional (separate) arity 2 body, and make the arity-1 one call it.